frontend/src/app/components/wp-single-view-tabs/additional-tab/additional-tab.html

```diff
@@ -0,0 +1,6 @@
+<div class="detail-panel-description detail-panel--integration detail-panel--autocomplete-target"
```

Comment:
TBH, I don't really know if we need all these classes, I copied them over from the other tabs and hope they are alright 🤞